test/filters/test_motion.py

```diff
@@ -0,0 +1,66 @@
+from typing import Tuple
+import pytest
+
+import kornia
+import kornia.testing as utils  # test utils
+from test.common import device_type
+
+import torch
+from torch.autograd import gradcheck
+from torch.testing import assert_allclose
+
+
+@pytest.mark.parametrize("ksize", [3, 11])
+@pytest.mark.parametrize("angle", [0., 360.])
+@pytest.mark.parametrize("direction", [-1., 1.])
+def test_get_motion_kernel2d(ksize, angle, direction):
+    kernel = kornia.get_motion_kernel2d((ksize, ksize), angle, direction)
+    assert kernel.shape == (ksize, ksize)
+    assert kernel.sum().item() == pytest.approx(1.0)
```

Comment:
use `assert_allclose`

Reply:
I made a fix. Not 100% sure if it was what you were meaning